README.md

```diff
@@ -463,3 +463,5 @@ the IDA Pro installation directory (`%ProgramFiles(x86)%\IDA 6.9` by default):
 
 The alternative would be to build a static PostgreSQL client library from
 source.
+
+If you are running binexport under Wine, libpq.dll requires that you install libldap (32-bit) in your distro (e.g. [lib32-libldap](https://www.archlinux.org/packages/multilib/x86_64/lib32-libldap/)).
```

Comment:
Yeah, without it libpq.dll fails to load. Sounds believable since
PostgresSQL deals with users.